gnomad_qc/v3/sample_qc/sample_qc.py

```diff
@@ -468,12 +468,29 @@ def run_pca(
     return run_pca_with_relateds(qc_mt, samples_to_drop, n_pcs=n_pcs)
 
 
+def annotate_subpop_meta(ht: hl.Table) -> hl.Table:
+    """
+    Add subpop_description to a Table.
+
+    :param ht: Table to which metadata annotation of subpop_description should be addded
+    :return: Table with added annotation for subpop_description
+    """
+    meta_ht = meta.ht()
+    ht = ht.annotate(**meta_ht[ht.key])
+    ht = ht.annotate(subpop_description=ht.project_meta.subpop_description)
```

Comment:
Sorry, one more. This is used like this below:
`pop_pca_scores_ht.project_meta.subpop_description`

So we really only need the meta_ht annotated on, I actually vote we just remove this function and do 

```
    meta_ht = meta.ht()
    ht = ht.annotate(**meta_ht[ht.key])
```

right in the function that uses `subpop_description`

```suggestion
```